Resources/Prototypes/Corvax/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```
@@ -0,0 +1,13 @@
+  id: PrinterDocMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: document printer machine board
+  description: A machine printed circuit board for an document printer
+  components:
+    - type: MachineBoard
+      prototype: PrinterDoc
+      requirements:
+        MatterBin: 3
+        Manipulator: 1
+      materialRequirements:
+        Glass: 1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)